.github/workflows/release-new-version.yml

```diff
@@ -0,0 +1,94 @@
+        env:
+          OSSRH_TOKEN_USERNAME: ${{ secrets.OSSRH_TOKEN_USERNAME }}
+          OSSRH_TOKEN_PASSWORD: ${{ secrets.OSSRH_TOKEN_PASSWORD }}
+          PGP_KEY_PASSPHRASE: ${{ secrets.JAVA_NATIVE_PGP_KEY_PASSPHRASE }}
+      - run: echo "releaseTag=$(git tag --points-at HEAD^)" >> $GITHUB_ENV
+      - run: echo "releaseVersion=$(echo ${{ env.releaseTag }} | cut -c2-)" >> $GITHUB_ENV
+      - name: Push changes back to repo
+        run: git push && git push --tags --force
+    outputs:
+      releaseTag: ${{ env.releaseTag }}
+      releaseVersion: ${{ env.releaseVersion }}
+
+  github-release:
+    runs-on: ubuntu-latest
+    needs: maven-central-release
+    steps:
+      - uses: actions/checkout@v2
+        with:
+          ref: ${{ needs.maven-central-release.outputs.releaseTag }}
+      - run: git config --global user.email "${GITHUB_BOT_EMAIL}"
+      - run: git config --global user.name "${GITHUB_BOT_NAME}"
+      - name: Set up JDK 11
+        uses: actions/setup-java@v2
+        with:
+          java-version: 11
+          distribution: temurin
+      - name: Build jar with dependencies included
+        run: mvn -B install -Ppackage,jar-with-dependencies
+      - name: Publish as release
+        uses: ncipollo/release-action@v1
+        with:
+          name: ${{ needs.maven-central-release.outputs.releaseVersion }}
+          tag: ${{ needs.maven-central-release.outputs.releaseTag }}
+          artifacts: target/*.jar
+          token: ${{ secrets.GITHUB_TOKEN }}
+
+  update-version-references:
+    runs-on: ubuntu-latest
+    needs: maven-central-release
+    env:
+      ver: ${{ needs.maven-central-release.outputs.releaseVersion }}
+    steps:
+      - uses: actions/checkout@v2
+      - run: |
+            git config --global user.email "${GITHUB_BOT_EMAIL}"
+            git config --global user.name "${GITHUB_BOT_NAME}"
+      - run: |
+            sed -i "/<artifactId>jssc<\/artifactId>/{n;s/<version>.*<\/version>/<version>${ver}<\/version>/}" README.md
+            sed -i "s/io.github.java-native:jssc:.*\"/io.github.java-native:jssc:${ver}\"/" README.md
+            sed -i "s/io.github.java-native:jssc:.*'/io.github.java-native:jssc:${ver}'/" README.md
+            git diff --quiet README.md || git commit -m "Update README.md" README.md && git pull --rebase && git push
+      - run: |
+            MAJOR=$(echo ${ver} | rev | cut -d. -f2- | rev)
+            MINOR=$(echo ${ver} | rev | cut -d. -f1 | rev)
+            FILE=src/main/java/jssc/SerialNativeInterface.java
+            sed -i "s/private static final String libVersion =.*/private static final String libVersion = \"${MAJOR}\";/" ${FILE}
+            sed -i "s/private static final String libMinorSuffix =.*/private static final String libMinorSuffix = \"${MINOR}\";/" ${FILE}
+            git diff --quiet ${FILE} || git commit -m "Update SerialNativeInterface" ${FILE} && git pull --rebase && git push
```

Comment:
Two concerns:
1. We should fix `SerialNativeInterface.java` directly instead of splitting up version information.
2. I feel that two commits (one for `README.md`, another for `SerialNativeInterface.java`) for a single version "bump" is overkill.  Do you agree?

Reply:
Removed automatic update of SerialNativeInterface from post release cleanup as I realized I used flawed reasoning and replaced the file with version that just got released, not a new one. 

Also, the version in this file should be set when building the native libraries (and as such may not even need bumping for certain releases with java only fixes) - so I will try to incorporate it into the other PR #102.

Reply:
As an aside, if it makes sense -- from an order-of-operations perspective -- to start versioning the native libraries separate from the release, I do not mind.  I'm not sure if this makes sense to do, but I wanted to mention it.